wordle_solver/multi_solver.py

```diff
@@ -0,0 +1,23 @@
+from .wordle_solver import WordleSolver
+
+
+class MultiSolver:
+    def __init__(self, keyboard_count: int, words_length: int, language: str):
+        self.solvers: list[WordleSolver] = [WordleSolver(words_length, language) for _ in range(keyboard_count)]
+
+    def reset(self):
+        for solver in self.solvers:
+            solver.reset()
+
+    def valid_word(self, index: int, word: str) -> bool:
```

Comment:
May want to make a method with no index parameter, that checks all of the solvers (AND gate).

Reply:
Maybe an OR gate, since as a player I may have enough clues from my previous guesses so that I won't have any word that is a possible solution to all of the subgames, but I would like to use words that are a possible solution to at least one of the subgames.

Reply:
True, an OR gate sounds right.
Should I open an issue for that?